File: 백준/Silver/11650. 좌표 정렬하기/좌표 정렬하기.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

const num = Number(input.shift())
const arr = input.map(v => v.split(' ').map(x => Number(x)))

let result = '';

arr.sort((a,b) => {
    if(a[0] !== b[0]) {
        return a[0] - b[0];
    } else {
        return a[1] - b[1];
    }
}).forEach(x => {
    result += `${x[0]} ${x[1]}\n`
})

console.log(result);

File: 백준/Silver/11650. 좌표 정렬하기/좌표 정렬하기.py
import sys

n = int(sys.stdin.readline())
arr = []
for i in range(n):
    a, b = (map(int, sys.stdin.readline().split()))
    arr.append((a, b))

arr.sort(key= lambda x:(x[0], x[1]))

for i in arr:
    print(i[0], i[1])